src/System.Private.ServiceModel/tests/Scenarios/Security/TransportSecurity/Tcp/IdentityTests.cs

```diff
@@ -0,0 +1,44 @@
+﻿using System;
+using System.ServiceModel;
+using System.Text;
+using Xunit;
+
+namespace Security.TransportSecurity.Tests.Tcp
+{
+    public static class IdentityTests
+    {
+        [Fact]
+        [ActiveIssue(12)]
+        [OuterLoop]
+        //Once this feature is enabled, we will need to add negative tests
+        public static void VerifyServiceDNS()
```

Comment:
NIT: Consider naming something like Verify_ServerValidationMode_ServerCertificate or something like that, so we have the binding type in the name. Otherwise, it's not super obvious what's being tested because the test just looks like any other echo test, and we don't explicitly validate the DnsEndpointIdentity aside from waiting to see if an exception is thrown.